pandas/lib.pyx

```diff
@@ -26,9 +26,9 @@ from cpython cimport (PyDict_New, PyDict_GetItem, PyDict_SetItem,
                       PyBytes_GET_SIZE,
                       PyUnicode_GET_SIZE)
 
-try:
+IF PY_MAJOR_VERSION == 2:
```

Comment:
@chris-b1 I could not get that to work since this is needed at compile time, so I needed the upper-case ``IF <env_var>``, not a runtime ``if <value>`` :( I reverted to the original fix